topic_reader.py

```diff
@@ -0,0 +1,218 @@
+# Calculate the read speed of Red Hat Training adoc files.
+  
+
+import os
+import math
+import sys
+import glob
+
+from configparser import ConfigParser
+
+import oyaml as yaml
+
+##### These are pulled from config.ini #####
+
+#The average reading speed in words per minute for adults
+lecture_read_speed = 0
+ge_read_speed = 0
+#The time to "read" each image in seconds
+image_read_time = 0
+#The time it takes to read a code block in seconds
+code_block_read_time = 0
+
+quiz_time = 0
```

Comment:
we can just delete this if using a config file, right?

Reply:
yes, done